rpcserver/source/rpcserver.cc

```diff
@@ -13,12 +15,17 @@ RPCServer::RPCServer(int p) : port{p} {}
  * Return true on success, false on failure. On success, version, service,
  * and auth protocol set from the received
  */
-bool RPCServer::receive_handshake(tcp::socket &sock, short *version, short *service, short *auth_protocol) {
+bool RPCServer::receive_handshake(
+    tcp::socket &sock,
+    int16_t *version,
+    int16_t *service,
+    int16_t *auth_protocol
+) {
   // handshake header.
   // Handshake has 7 bytes.
-  constexpr size_t handshake_len = 7;
-  char data[handshake_len];
-  asio::error_code error = read_full(sock, asio::buffer(data, handshake_len));
+  constexpr size_t kHandshakeLen = 7;
```

Comment:
What is the motivation for this change?

Reply:
there was a lint error saying it should be named like kSomething since it's a constant